src/addUpdateEmployee.ts

```diff
@@ -0,0 +1,326 @@
+    dataEntrySkillAlert.style.display = "block";
+  } else dataEntrySkillAlert.style.display = "none";
+  if (!email.checkValidity()) {
+    dataEntryEmailAlert.style.display = "block";
+  } else dataEntryEmailAlert.style.display = "none";
+
+  if (dataEntrySubmit.value == "Add") {
+    if (!isErr) {
+      let entryIndex = 0;
+      if (actualData.employee) {
+        actualData.employee.forEach((elem, index) => {
+          if (index > entryIndex) entryIndex = index;
+        });
+        entryIndex = entryIndex + 1;
+      } else entryIndex = 0;
+
+      let employeeID = 1001;
+      if (originalData.employee) {
+        employeeID =
+          Number(originalData.employee[originalData.employee.length - 1].id) +
+          1;
+      } else employeeID = 1001;
+
+      let errMsg = "Error while adding employee";
+      let succMsg = "Succesfully added employee";
+      putData(
+        entryIndex,
+        employeeID,
+        nameVal,
+        emailVal,
+        dateOfJoinVal,
+        dateOfBirthVal,
+        depInputVal,
+        roleInputVal,
+        locInputVal,
+        skillInputVal,
+        base64String,
+        errMsg,
+        succMsg
+      );
+      console.log(employeeID, "employee id that is going for the new data");
+      console.log(entryIndex, "index that the new data occupies");
+      dataEntryForm.reset();
+      addedSkills.innerHTML = "";
+      //   skillNameArr = [];
+      let nullArr: string[] = [];
+      changeSkillNameArr(nullArr);
+      overlay.style.display = "none";
+      dataEntryModal.style.display = "none";
+    } else {
+      console.log("error in adding new employee");
+    }
+  } else if (dataEntrySubmit.value == "Update") {
+    if (!isErr) {
+      let employeeID = idOfEmp;
+
+      base64String = originalData.employee[updateIndex].imageSrc;
+
+      let errMsg = "Error while updating employee";
+      let succMsg = "Succesfully updated employee";
+      putData(
+        updateIndex,
+        employeeID,
+        nameVal,
+        emailVal,
+        dateOfJoinVal,
+        dateOfBirthVal,
+        depInputVal,
+        roleInputVal,
+        locInputVal,
+        skillInputVal,
+        base64String,
+        errMsg,
+        succMsg
+      );
+      console.log(employeeID, "id that is going to be updated");
+      console.log(updateIndex, "index that is going to be updated");
+
+      overlay.style.display = "none";
+      dataEntryModal.style.display = "none";
+    }
+  }
+};
+dataEntryForm.onsubmit = handleSubmitClick;
+
+//data-entry-form skill section functionalities
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+Fulltable.onclick = (e: MouseEvent) => {
+  const target = e.target as HTMLElement;
+  if (target.classList.contains("edit-image-icon")) {
+    // skillNameArr = skillName;
+    changeSkillNameArr(skillName);
+  }
+};
+
+formSkill.onclick = (e: MouseEvent) => {
+  const target = e.target as HTMLElement;
+  // for chrome
```

Comment:
remove commented code - Adding commented descriptions as text is fine but don't add commented code